package.json

```diff
@@ -36,6 +36,7 @@
         "rollup-plugin-copy": "^3.4.0",
         "rollup-plugin-typescript2": "^0.31.1",
         "ts-jest": "^27.0.7",
+        "tsc-alias": "^1.4.2",
```

Comment:
@barisbora you know if there is a rollup plugin for this?

Reply:
I saw somewhere, I'll check